node/src/executor.rs

```diff
@@ -47,9 +46,9 @@ mod tests {
 		NativeOrEncoded
 	};
 	use node_primitives::{Hash, BlockNumber, AccountId, Balance, Index};
-	use runtime_primitives::traits::{Header as HeaderT, Hash as HashT};
-	use runtime_primitives::{generic::Era, ApplyOutcome, ApplyError, ApplyResult, Perbill};
-	use runtime_primitives::weights::{WeightMultiplier, SimpleDispatchInfo, WeighData};
+	use sr_primitives::traits::{Header as HeaderT, Hash as HashT};
```

Comment:
I am quite happy to see some else also changing these --not wrong but confusing-- imports; it is quite annoying that we have `core/sr-primitives` imported into different crates with different names; makes it even more confusing that we have an actual `runtime-primitives` in `node`.

I have some thoughts on _codifying_ imports a bit and adding it to our style guide maybe. Will make an issue for it.